src/sphinxcontrib/towncrier/_version.py

```diff
@@ -1,6 +1,7 @@
 """Version definition."""
 
 try:
+    # pylint: disable=unused-import
     from ._scm_version import version as __version__
```

Comment:
It's preferable to put the ignores on the same line they are applied against:
```suggestion
    from ._scm_version import version as __version__  # pylint: disable=unused-import
```

Reply:
Oh, it'd be too long. Nevermind, then...